apps/prairielearn/src/lib/config.ts

```diff
@@ -24,9 +26,21 @@ const ConfigSchema = z.object({
       // https://node-postgres.com/features/ssl
       z.object({
         rejectUnauthorized: z.boolean().default(true),
-        ca: z.string().nullable().default(null),
-        key: z.string().nullable().default(null),
-        cert: z.string().nullable().default(null),
+        ca: z
+          .string()
+          .nullish()
+          .default(null)
```

Comment:
Is the default here necessary?

Reply:
Nope; removed in 5df38da. I also added `.strict()` to this object to reject unknown properties, which should be valuable for configuration where one might make typos.